src/Configuration/ValidationExtensions.cs

```diff
@@ -0,0 +1,111 @@
+﻿/*
+ * Apache License, Version 2.0
+ * Copyright 2019-2021 NVIDIA Corporation
```

Comment:
is this all new code? if so the copyright should be 2021 and not 2019-2021

Reply:
Code refactored from another file.